src/_includes/docs/swift-package-manager/migrate-macos-project-manually.md

```diff
@@ -0,0 +1,74 @@
+Once you enable the Swift Package Manager feature, the Flutter CLI will try
+to migrate your project to add Swift Package Manager integration. This makes
+your project download the Swift packages that your Flutter plugins depend on.
+
+However, the Flutter CLI may be unable to migrate your project if you've
```

Comment:
```suggestion
However, the Flutter CLI might be unable to migrate your project if you've
```

You might wonder why I keep changing "may" to "might". It's in our wordlist. "May" is generally used in legal contexts.  https://developers.google.com/style/word-list#may

Reply:
Thanks for the link, that's a good resource! :)